configs/openeo.json

```diff
@@ -1,30 +1,88 @@
 {
```

Comment:
Same comments about `page_rank: 0` and `start_urls` order

Reply:
Fixed